index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Personal Portfolio</title>
+        <link href="./css/flexbox.css" rel="stylesheet" type="text/css">
+        <link href="./css/index.css" rel="stylesheet" type="text/css">
+    </head>
+    <body>
+        <header>
+          <h1>
+            Srey Mom Pech
+          </h1>
+        </header>
+        <nav>
```

Comment:
Nice! A navbar is commonly implemented with an unordered list within a `<nav>` 👍🏻